roles/klaytn_node/tasks/download_homi_tool.yml

```diff
@@ -26,6 +42,18 @@
     static_nodes_json: []
     static_nodes_knis: []
   tags: "launchhomi"
+  when:
+    - ansible_os_family != "Darwin"
```

Comment:
What about using include by ansible_os_family
```
- include: debian.yml
  when: ansible_os_family == 'Debian'

- include: redhat.yml
  when: ansible_os_family == 'RedHat'
```
ref : [stackoverflow](https://stackoverflow.com/questions/33762738/specifying-the-os-ansible)

Reply:
@henry-will Thank you for the comment.
Currently, the [Klaytn package](https://docs.klaytn.com/node/download) supports **Linux** and **MacOS**. Also, the [System requirements for Endpoint Node](https://packages.klaytn.net/klaytn/v1.8.2/homi-v1.8.2-0-linux-amd64.tar.gz) in Klaytn Docs states that Klaytn should work on linux-based environments. Though I have never tested on other OS, I assumed that there can be other distributions that Klaytn can run on. So I decided to categorize OS into two group: Linux and MacOS like below:
```
# Check OS type
- name: "Check hosts' OS type"
  fail: msg="The hosts' OS should be either Linux or MacOS."
  when:
    - ansible_system != "Linux"
    - ansible_os_family != "Darwin"
```

Still, should I limit target OS to Debian and Redhat?

Reply:
I am sorry to confuse you. Debian and Redhat is just examples. I want to use "- include:" feature. Only Linux and Darwin is good to me. I suggest that homi_linux.yml, homi_darwin.yml, gen_tool_linux.yml and gen_tool_darwin.yml files make separately.

Reply:
Oh, I see.
Then, I have another question. Only a portion of the tasks requires different actions for different OS. Do you want to separate only the distinct actions to different files, or do you think it is OK to have duplicate actions in different files?

For example, `download_homi_tool.yml` not only downloads different files for different OS, but it also runs `homi` to generate configuration files for Klaytn. The actions for running the homi are independent from the OS type. Then, which one do you prefer?
1. Separate only the OS-dependent actions, e.g., `download_homi_linux.yml`, `download_homi_macos.yml`, and use another file to perform OS-independent actions, e.g., `run_homi.yml`. This option may result in too many task files with some files only containing a few (2~3) actions.
2. Separate into two files for a set of related actions, e.g., `download_and_run_homi_linux.yml` and `download_and_run_homi_macos.yml`. This option creates duplicated actions in the files for Linux and MacOS.

Reply:
I like number 1 case, because common case is only used one file. I think that It is for managing code and maintenance.

Reply:
I will handle this suggestion in another PR, after other PRs are merged.